src/components/Table/Table.test.jsx

```diff
@@ -742,16 +739,16 @@ describe("Table", () => {
 			expect(menuItems).toHaveLength(4);
 			await user.click(queryAllByRole("menuitem")[3]);
 
-			expect(getByRole("dialog")).toHaveClass("sheet-horizontal-slide-in-shim");
-			expect(getByRole("dialog")).not.toHaveClass(
-				"sheet-horizontal-slide-out-shim",
-			);
+			expect(getByRole("dialog")).toHaveClass("neo-drawer neo-drawer--isOpen");
 
 			const nameCheckbox = getByLabelText(FilledFields.columns[0].Header);
 			expect(nameCheckbox).toBeChecked();
 
 			await user.click(nameCheckbox);
 			expect(nameCheckbox).not.toBeChecked();
+
+			const applyButton = getByLabelText("Apply");
```

Comment:
Don't use static strings as these are fragile and tests will be broken if the translation data is updated (which has happened numerous times as Matt+Justin+David ask for translation updates frequently). Instead, use `FilledFields.translations.*`. Ie:
```suggestion
			const applyButton = getByLabelText(FilledFields.translations.toolbar.apply);
```

Reply:
the same on line 784 (`const applyButton = getByLabelText("Apply");`).

Reply:
of course. fixed.